### ISSUE TITLE
Error 500

### ISSUE THREAD

**Issue by nitrosx** [silx-kit/jupyterlab-h5web#106]
### Describe the bug
When running your example.ipynb, I get the following error:
```python
from jupyterlab_h5web import H5Web

H5Web('simple.h5')

Request failed with status code 500
```

### To Reproduce
1. run jupyter lab
2. open the notebook example.ipynb
3. run the first two cells

### Expected behaviour
It should present the widget

### Context
- OS: LinuxMint
- Browser: Firefox
- Version: 110 64bits
- JupyterLab version: 3.5.3
- Python: 3.10
- conda environment

### Extension lists
```bash
> jupyter labextension list
JupyterLab v3.5.3
/home/nitrosx71/miniconda3/envs/SE/share/jupyter/labextensions
        jupyterlab-h5web v7.0.0 enabled OK (python, jupyterlab_h5web)
```

```bash
> jupyter serverextension list
config dir: /home/nitrosx71/miniconda3/envs/SE/etc/jupyter
    jupyterlab  enabled 
    - Validating...
      jupyterlab 3.5.3 OK
    jupyterlab_h5web  enabled 
    - Validating...
      jupyterlab_h5web 7.0.0 OK
```

**Comment by loichuder** [comment]
Hm, that's odd. It works fine on my end with the same version of `jupyterlab`.

Do you open JupyterLab by running `jupyter lab` in a terminal ? If so, do you have additional info in the log displayed in the terminal about the reason of the error 500 ?

**Comment by yurirtonin** [comment]
![Screenshot 2023-05-12 at 16 01 39](https://github.com/silx-kit/jupyterlab-h5web/assets/16940168/2398bf4a-b037-475a-9dec-a9eb0512a0eb)

I had a similar issue with a simple dummy file I created. The terminal log complained about the HDF version.

**Comment by t20100** [comment]
It's related to the version of libhdf5 in used that does not support the "file locking" feature:
```
locking – The file locking behavior. One of:
[...]
Only available with HDF5 >= 1.12.1 or 1.10.x >= 1.10.7.
```
See https://docs.h5py.org/en/stable/high/file.html#h5py.File

You can check the version of libhdf5 with `h5py.version.hdf5_version`.
